### PR TITLE
Stability and releases

### PR DESCRIPTION
**Pull request by bartlomieju**
First pass, feedback welcome.

Closes https://github.com/denoland/docs/issues/886

### CODE REVIEW

File: runtime/fundamentals/stability_and_releases.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+When introducing new APIs, these are first marked as unstable. This means that
+the API may change in the future. These APIs are not available to use unless you
+explicitly pass an unstable flag, like `--unstable-kv`.
+[Learn more about `--unstable-*` flags](/runtime/reference/cli/unstable_flags).
+
+There are also some non-runtime features of Deno that are considered unstable,
+and are locked behind unstable flags. For example, the
+`--unstable-sloppy-imports` flag is used to enable `import`ing code without
+specifying file extensions.
```
Comment (ry) [inline runtime/fundamentals/stability_and_releases.md:19]: put this in a section `## Unstable APIs` and move it to the bottom

File: runtime/fundamentals/stability_and_releases.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Release schedule, channels and long term support
+
+Deno releases a new stable, minor version (eg. v2.1.0, v2.0.0) on a monthly
+schedule.
```
Comment (ry) [inline runtime/fundamentals/stability_and_releases.md:24]: Could add ...mostly inline with Chrome V8 releases

File: runtime/fundamentals/stability_and_releases.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+LTS eligible backports include security fixes and defect fixes (crashes, wrong
+answers).
+
+Critical performance improvements **may** be backported based on severity.
+
+API changes and major new features **will not** be backported.
+
+## Standard library
+
+The Deno Standard Library (https://jsr.io/@std) is partially stable. All
```
Comment (ry) [inline runtime/fundamentals/stability_and_releases.md:66]: s/partially/mostly

File: runtime/fundamentals/stability_and_releases.md (hunk 2 above)
Comment (bartlomieju) [reply runtime/fundamentals/stability_and_releases.md:24]: I think this information is irrelevant to an average Deno user.

File: runtime/fundamentals/stability_and_releases.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+- `stable` - a semver minor/patch release, as described above. This is **the
+  default** distribution channel that is recommended for most users.
+- `lts` - long term support for a particular stable release, recommended for
+  enterprise users who prefer not to upgrade so often. See below for details.
+- `rc` - a release candidate for the upcoming semver minor release.
+- `canary` - an unstable release that changes multiple timer per day, allows to
+  try out latest bug fixes and new features that might end up in the `stable`
+  channel.
+
+### LTS releases
```
Comment (ry) [inline runtime/fundamentals/stability_and_releases.md:33]: `## Long Term Support (LTS)`

**Review by ry — APPROVED** [review]
LGTM

**Pull request merged**